tests/core/test_utilities.py

```diff
@@ -1370,38 +1374,48 @@ def test_transform_invert(transform):
     assert transform.is_inverted is False
 
 
-@pytest.mark.parametrize('copy', [True, False])
-@pytest.mark.parametrize(
-    ('obj', 'return_self', 'return_type', 'return_dtype'),
-    [
-        (list(VECTOR), False, np.ndarray, float),
-        (VECTOR, False, np.ndarray, float),
-        (np.array(VECTOR), False, np.ndarray, float),
-        (np.array([VECTOR]), False, np.ndarray, float),
-        (np.array(VECTOR, dtype=float), True, np.ndarray, float),
-        (np.array([VECTOR], dtype=float), True, np.ndarray, float),
-        (pv.PolyData(np.atleast_2d(VECTOR)), True, pv.PolyData, np.float32),
-        (pv.PolyData(np.atleast_2d(VECTOR).astype(int)), True, pv.PolyData, np.float32),
-        (pv.PolyData(np.atleast_2d(VECTOR).astype(float)), True, pv.PolyData, float),
-        (
+class Cases_transform_apply:
```

Comment:
nit: I don't think we have a documented convention but this formatted is more consistent with our class definitions
```suggestion
class CasesTransformApply:
```